scripts/exgdas_global_aero_analysis_finalize.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+################################################################################
+#  UNIX Script Documentation Block
+#                      .                                             .
+# Script name:         exgdas_global_aero_analysis_finalize.py
+# Script description:  Finalizes runtime directory for global aerosol analysis
+#
+# Author: Cory Martin      Org: NCEP/EMC     Date: 2022-8-29
+#
+# Abstract: This script cleans up the runtime directory and moves/archives
+#           necessary output files ffrom FV3-JEDI executable(s)
+#           produced from a UFS Global Aerosol Analysis.
+#
+# $Id$
+#
+# Attributes:
+#   Language: Python3
+#
+################################################################################
+
+# import os and sys to add ush to path
+import logging
+import os
+import shutil
+import sys
+
+# set up logger
+logging.basicConfig(format='%(asctime)s:%(levelname)s:%(message)s', level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
+
+# get absolute path of ush/ directory either from env or relative to this file
+my_dir = os.path.dirname(__file__)
+my_home = os.path.dirname(os.path.dirname(my_dir))
+sys.path.append(os.path.join(os.getenv('HOMEgfs', my_home), 'ush'))
+logging.info(f"sys.path={sys.path}")
```

Comment:
This is to get the ush directory on the python path for the import below, the other approaches would be to 'install' or to add it at the j-job step, unless there is another alternative?

Reply:
We may need to just add some path (either `ush` or `ush/pygw`) to the python path for all jobs. I'll defer to @aerorahul on what he wants to do with this, because it is an issue that is going to keep cropping up.